src/recorder.js

```diff
@@ -148,7 +149,13 @@ var root = (typeof self === 'object' && self.self === self && self) || (typeof g
       var that = this;
       this.encoder = new global.Worker( this.config.encoderPath );
 
-      if (this.config.streamPages){
+      if (this.config.rawPacket){
```

Comment:
Less code duplication if we did `if (this.config.rawPacket || this.config.streamPages) { ...` Also please add a note this behaviour in the readme.